tests/support/server/server.coffee

```diff
@@ -4,7 +4,13 @@ wss = new WebSocketServer(port: CONFIG.REALLY_PORT)
 
 wss.on 'connection', (ws) ->
   ws.on 'message', (message) ->
-    ws.send message
-    console.log "Received #{message}"
+    msg = JSON.parse message
+    if msg.cmd is 'error'
```

Comment:
we don't want to cause conflicts with our protocol we already have key name `cmd` why didn't we just craft a new one for tests to avoid possible conflicts